op-chain-ops/script/script.go

```diff
@@ -697,7 +687,9 @@ func (h *Host) StateDump() (*foundry.ForgeAllocs, error) {
 	baseState := h.baseState
 	// We have to commit the existing state to the trie,
 	// for all the state-changes to be captured by the trie iterator.
-	root, err := baseState.Commit(h.env.Context.BlockNumber.Uint64(), true)
+	// TODO: do we need to check `IsCancun()` here, or can we just pass false?
+	isCancun := h.chainCfg.IsCancun(h.env.Context.BlockNumber, h.env.Context.Time)
+	root, err := baseState.Commit(h.env.Context.BlockNumber.Uint64(), true, isCancun)
```

Comment:
TBD

Reply:
Just using `false` here is fine, we assume cancun is active everywhere here, and there's no historical sync in the script host environment.